user-guide/manual.adoc

```diff
@@ -115,22 +119,107 @@ TBD
 |BSL Source[[bsl-source]]
 |https://github.com/NASA-AMMOS/BSL[bsl-source]
 
-|BSL Document Source[[bsl-docs]]
+|BSL Documentation Source[[bsl-docs]]
 |https://github.com/NASA-AMMOS/BSL-docs[bsl-docs]
 
+|Bundle Protocol Version 7[[RFC9171]]
+|https://www.rfc-editor.org/info/rfc9171
+
+|Bundle Protocol Security (BPSec)[[RFC9172]]
+|https://www.rfc-editor.org/info/rfc9172
+
+|Default Security Contexts for Bundle Protocol Security (BPSec)[[RFC9173]]
+|https://www.rfc-editor.org/info/rfc9173
+
 |===
 
 
 == Application Programming Interface
 
 The following section provides an overview of the BSL API and references to specific sections of the online API documentation.
 
-TBD
+=== Introduction
```

Comment:
The unnumbered first section is already titled "Introduction" so maybe this can be "Architecture" section...?